packages/web/src/store/account/sagas.js

```diff
@@ -69,6 +69,25 @@ import mobileSagas, { setHasSignedInOnMobile } from './mobileSagas'
 
 const NATIVE_MOBILE = process.env.REACT_APP_NATIVE_MOBILE
 
+const IP_STORAGE_KEY = 'user-ip-timestamp'
+
+function* recordIPIfNotRecent() {
+  const twentyFourHours = 24 * 60 * 60 * 1000
```

Comment:
nit: probably shouldn't name this variable the duration of the interval itself, because that means we can't change the interval without changing the variable name